src/bandmap.c

```diff
@@ -413,9 +418,11 @@ void bandmap_age() {
 	spot *data = list->data;
 	GList *temp = list;
 	list = list->next;
-	if (data->timeout) {
-	    data->timeout--;
-	    if (data->timeout == 0) {
+	if (data->timeout >= 0) {
+	    if (bmautograb == 0 || (bmautograb > 0 && data->timeout > 0)) {
```

Comment:
The whole logic here seems weird. If bmautograb is 0 then the timeout will be decremented even if it ist already 0. That makes no sense.